tests/testing_mesh_creation_functions.py

```diff
@@ -737,6 +738,99 @@ def line(t):
         # Check the output.
         compare_test_result(self, input_file.get_string(header=False))
 
+    def test_mesh_creation_functions_helix_no_rotation(self):
+        """Create a helix and compare it with the reference file."""
+
+        # Create mesh.
+        mesh = Mesh()
+
+        # Create input file.
+        input_file = InputFile()
+
+        # Add material and function.
+        mat = MaterialReissner(youngs_modulus=1e5, radius=0.5, shear_correction=1.0)
+
+        # Create helix.
+        create_beam_mesh_helix(
+            mesh,
+            Beam3rHerm2Line3,
+            mat,
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 0.0],
+            [2.0, 0.0, 0.0],
+            np.pi / 4,
+            height_helix=80.0,
+            l_el=5.0,
+        )
+
+        # Add mesh.
+        input_file.add_mesh(mesh)
+
+        # Check the output.
+        compare_test_result(self, input_file.get_string(header=False))
+
+    def test_mesh_creation_functions_helix_rotation_offset(self):
+        """Create a helix and compare it with the reference file."""
+
+        # Create mesh.
+        mesh = Mesh()
+
+        # Create input file.
+        input_file = InputFile()
+
+        # Add material and function.
+        mat = MaterialReissner(youngs_modulus=1e5, radius=0.5, shear_correction=1.0)
+
+        # Create helix.
+        create_beam_mesh_helix(
+            mesh,
+            Beam3rHerm2Line3,
+            mat,
+            [1.0, 1.0, 1.0],
+            [-1.0, -1.0, -1.0],
+            [3.0, 0.0, 0.0],
+            np.pi / 6,
+            height_helix=80.0,
+            l_el=5.0,
+        )
+
+        # Add mesh.
+        input_file.add_mesh(mesh)
+
+        # Check the output.
+        compare_test_result(self, input_file.get_string(header=False))
+
+    def test_mesh_creation_functions_helix_radius_zero(self):
+        """Create a helix and compare it with the reference file."""
+
+        # Create mesh.
+        mesh = Mesh()
+
+        # Create input file.
+        input_file = InputFile()
+
+        # Add material and function.
+        mat = MaterialReissner(youngs_modulus=1e5, radius=0.5, shear_correction=1.0)
+
+        # Create helix.
+        create_beam_mesh_helix(
+            mesh,
+            Beam3rHerm2Line3,
+            mat,
+            [1.0, 1.0, 1.0],
+            [-1.0, -1.0, -1.0],
+            [1.0, 1.0, 1.0],
+            np.pi / 6,
+            height_helix=80.0,
+            n_el=10,
```

Comment:
You should use fewer elements for tests, 3-4 elements for the helix should be sufficient (also for the previous test cases).

Reply:
I've reduced all setups to 3-4 elements